tests/pkg/fixtures/cert_subject_mapping_queries.go

```diff
@@ -43,3 +46,27 @@ func CleanupCertificateSubjectMapping(t require.TestingT, ctx context.Context, g
 
 	return &result
 }
+
+func FindCertSubjectMappingForApplicationTemplate(t *testing.T, ctx context.Context, gqlClient *gcli.Client, appTemplateID string) *graphql.CertificateSubjectMapping {
+	after := ""
+	for {
+		queryCertSubjectMappingReq := FixQueryCertificateSubjectMappingsRequestWithPagination(300, after)
+		currentCertSubjectMappings := graphql.CertificateSubjectMappingPage{}
+
+		t.Log("Getting current certificate subject mappings...")
+		err := testctx.Tc.RunOperationWithoutTenant(ctx, gqlClient, queryCertSubjectMappingReq, &currentCertSubjectMappings)
+		require.NoError(t, err)
+
+		after = string(currentCertSubjectMappings.PageInfo.EndCursor)
+
+		for _, csm := range currentCertSubjectMappings.Data {
+			if str.PtrStrToStr(csm.InternalConsumerID) == appTemplateID {
```

Comment:
It's good to check other data from the CSM as well - subject, consumerType, tenantAccess, etc.